projects/plugins/jetpack/modules/protect/math-fallback.php

```diff
@@ -57,9 +57,9 @@ static function math_authenticate() {
 				Jetpack_Protect_Math_Authenticate::generate_math_page();
 			} elseif ( ! hash_equals( $salted_ans_1, $correct_ans ) && ! hash_equals( $salted_ans_2, $correct_ans ) ) {
 				wp_die(
-				__( '<strong>You failed to correctly answer the math problem.</strong>  This is used to combat spam when the Protect API is unavailable.  Please use your browser\'s back button to return to the login form, press the "refresh" button to generate a new math problem, and try to log in again.', 'jetpack' ),
-				'',
-				array ( 'response' => 401 )
+					esc_html__( '<strong>You failed to correctly answer the math problem.</strong> This is used to combat spam when the Protect API is unavailable. Please use your browser\'s back button to return to the login form, press the "refresh" button to generate a new math problem, and try to log in again.', 'jetpack' ),
```

Comment:
This will not work without a change I'm afraid. Since you've changed from `__()` to `esc_html__()`, the `strong` tag will now be escaped.

Reply:
Done :)